content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -262,9 +262,9 @@ In the above two cases of the `Guaranteed` pod, the `none` CPU Manager policy wo
 
 In the case of the `BestEffort` pod, the `static` CPU Manager policy would send back the default topology hint as there is no CPU request and the Device Manager would send back the hints for each of the requested devices.
 
-Using this information the Topology Manager calculates the optimal hint for the pod and stores this information, which will be used by the Hint Providers when they are making their resource assignments. 
+Using this information the Topology Manager calculates the optimal hint for the pod and stores this information, which will be used by the Hint Providers when they are making their resource assignments.
 
 ### Known Limitations
 1. The maximum number of NUMA nodes that Topology Manager allows is 8. With more than 8 NUMA nodes there will be a state explosion when trying to enumerate the possible NUMA affinities and generating their hints.
-
-2. The scheduler is not topology-aware, so it is possible to be scheduled on a node and then fail on the node due to the Topology Manager.
\ No newline at end of file
+2. The default scheduler is not topology-aware, it is possible to be scheduled on a node and then fail on the node due to the Topology Manager. However, [out-of-tree scheduler](https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/pkg/noderesourcetopology) scheduler plugin mitigates this problem and allows optimal scheduling by taking resource topology into consideration.
+In order to achieve this, the plugin needs to subscribe to NodeResourceTopology modification generated by Node Resource Topology Exporter which exposes resource information with NUMA node granularity. [Node Feature Discovery](https://github.com/kubernetes-sigs/node-feature-discovery) implements Node Resource Topology Exporter in one of its plugins.

```

Comment:
Can you clarify what a "`NodeResourceTopology` modification" is? Specifically around. the word "modification" -- what is meant by that?

Reply:
"NodeResourceTopology modification" here refers to the  NodeResourceTopology CRs which capture per numa resource information corresponding to the nodes in the cluster. 

I think we can do a better job at phrasing this, maybe we change it something like below:

In order to achieve this, the plugin needs to access NodeResourceTopology CRs generated by Node Resource Topology Exporter which exposes resource information with NUMA node.....